tritium-tracing/src/test/java/com/palantir/tritium/tracing/JavaTracingTracerTest.java

```diff
@@ -40,7 +40,7 @@ public void trace() {
                 .hasSize(1)
                 .first()
                 .extracting("operation")
-                .contains("test");
+                .isEqualTo("test");
```

Comment:
Per https://assertj.github.io/doc/#assertj-core-3-13-0-release-notes

> As the extracting(String) method for Object and Map extracts only one
> value, it now returns Object assertions instead of list assertions (on
> a singleton list). This means that any list assertions used won’t
> compile anymore, they need to be replaced by Object assertions.

Reply:
slick.